src/stt/src/flt/flute.cpp

```diff
@@ -1892,7 +1904,7 @@ void local_refinement(int deg, Tree *tp, int p) {
       tp->branch[index[ii]].y = tt.branch[ii].y;
       tp->branch[index[ii]].n = index[tt.branch[ii].n];
     }
-    free(tt.branch);
+    tt.branch.clear();
```

Comment:
despite what the extremely confused pdrev authors think, clear has nothing to do with free. Since the branch vector is owned by the stack allocated variable tt there is no point in clearing it.